packages/node/src/serverless-functions/helpers.ts

```diff
@@ -91,23 +63,43 @@ function redirect(
   return res;
 }
 
-function setCharset(type: string, charset: string) {
-  // eslint-disable-next-line @typescript-eslint/no-var-requires
-  const { parse, format } = require('content-type');
-  const parsed = parse(type);
-  parsed.parameters.charset = charset;
-  return format(parsed);
```

Comment:
Why was this function moved up a couple lines? Can we avoid moving functions to keep the diff smaller?